CMakeLists.txt

```diff
@@ -281,4 +288,7 @@ else()
     message(STATUS "unit tests disabled")
 endif()
 
-add_subdirectory(doc)
+if (NOT NGRAPH_DOC_BUILD_DISABLE)
```

Comment:
Should be called NGRAPH_DOC_BUILD_ENABLE to conform to all other flags

Reply:
Done.